lib/Drush.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drush.
+ */
+
+use Drupal\Core\DependencyInjection\ContainerNotInitializedException;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Static Service Container wrapper.
+ *
+ * This code is analagous to the \Drupal class in Drupal 8.
+ *
+ * We would like to move Drush towards the model of using constructor
+ * injection rather than globals. This class serves as a unified global
+ * accessor to arbitrary services for use by legacy Drush code.
+ *
+ * Advice from Drupal 8's 'Drupal' class:
+ *
+ * This class exists only to support legacy code that cannot be dependency
+ * injected. If your code needs it, consider refactoring it to be object
+ * oriented, if possible. When this is not possible, and your code is more
+ * than a few non-reusable lines, it is recommended to instantiate an object
+ * implementing the actual logic.
+ *
+ * @code
+ *   // Legacy procedural code.
+ *   $object = drush_get_context('DRUSH_CLASS_LABEL');
+ *
+ * Better:
+ *   $object = \Drush::service('label');
+ *
+ *   function hook_do_stuff() {
+ *     $lock = lock()->acquire('stuff_lock');
+ *     // ...
+ *   }
+ * @endcode
```

Comment:
Cut-copy-pasta error!  I based my comment on the advice given in Drupal 8, edited it for Drush, and did not remove that part. It should not be there.